web/src/signalsHandlers.js

```diff
@@ -55,6 +56,7 @@ const SignalType = {
   // both the archived goal, and everything connected to it that
   // has archived at the same time
   ArchiveGoalFully: 'archive_goal_fully',
+  EditingGoal: 'editing_goal',
```

Comment:
We might bump this out into its own category since we've broken into new territory. This is a state update that isn't specifically a CRUD related action for an entry type (GoalComment, GoalVote, etc.)

So as its own declaration: 
```javascript
const NonEntrySignalTypes = {
  EditingGoal: 'editing_goal'
}
```

then we need to check in a special case below for that, around line 133 in this file